Pipfile

```diff
@@ -0,0 +1,23 @@
+[[source]]
```

Comment:
Please don't check a Pipfile in. Requirements go in `requirements.txt` or `dev-requirements.txt` or occasionally `docs/requirements.txt`